packages/client/src/lib/resources.ts

```diff
@@ -1,3 +1,7 @@
-export const HTTP_URL = process.env.REACT_APP_HTTP_URL;
+import { http } from 'utils/rest';
```

Comment:
We might have to re-think the naming of these files, `resources` and `rest`, do you think they are correct?

Reply:
I don't mind too much. http it's just a name that make sense with http.get http.delete and so on, if you have a better name we can change it

Reply:
No... socket.io can potentially use http, but that's not its main usage. Fine with `http`.